--- a/src/gallium/targets/dri/meson.build
+++ b/src/gallium/targets/dri/meson.build
@@ -92,6 +92,7 @@
                'st7735r_dri.so',
                'stm_dri.so',
 	       'sun4i-drm_dri.so',
+	       'sunxi-drm_dri.so',
              ]],
              [with_gallium_radeonsi, 'radeonsi_dri.so'],
              [with_gallium_nouveau, 'nouveau_dri.so'],
--- a/src/gallium/targets/dri/target.c
+++ b/src/gallium/targets/dri/target.c
@@ -122,6 +122,7 @@
 DEFINE_LOADER_DRM_ENTRYPOINT(st7735r)
 DEFINE_LOADER_DRM_ENTRYPOINT(stm)
 DEFINE_LOADER_DRM_ENTRYPOINT(sun4i_drm)
+DEFINE_LOADER_DRM_ENTRYPOINT(sunxi_drm)
 #endif
 
 #if defined(GALLIUM_LIMA)
--- a/debian/rules
+++ b/debian/rules
@@ -90,7 +90,7 @@
 	# etnaviv, tegra, vc4 and v3d kernel support are only available on armhf and arm64
 	ifneq (,$(filter $(DEB_HOST_ARCH), armhf arm64))
 		GALLIUM_DRIVERS += etnaviv panfrost svga tegra vc4 v3d
-		VULKAN_DRIVERS += broadcom freedreno
+		VULKAN_DRIVERS += broadcom freedreno panfrost
 	endif
 
 	ifneq (,$(filter $(DEB_HOST_ARCH), armhf arm64 riscv64))
